src/components/oculus-touch-controls.js

```diff
@@ -150,6 +150,7 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
       controller: 0,
       rotationOffset: data.rotationOffset !== -999 ? data.rotationOffset : isRightHand ? -90 : 90
     });
+    el.emit('controller-connected', {type: 'oculus-touch-controller'});
```

Comment:
this is why I asked about moving the event to `tracked-controls`. `tracked-controls` will pass the Gamepad ID and `el` as details of the event. @fernandojsg would that work for you?